.circleci/config.yml

```diff
@@ -254,6 +254,25 @@ jobs:
           path: /tmp/test-reports
       - store_artifacts:
           path: /tmp/test-artifacts/conformance-coverage.html
+  test-lotus-soup:
+    description: |
+      Compile `lotus-soup` Testground test plan using the current version of Lotus.
+    parameters:
+      <<: *test-params
+    executor: << parameters.executor >>
+    steps:
+      - install-deps
+      - prepare
+      - run: cd extern/oni && git submodule sync
+      - run: cd extern/oni && git submodule update --init
+      - run: cd extern/oni/extra/filecoin-ffi && make
```

Comment:
ℹ️ This is running the oni tests against the version of FFI that is submoduled in ONI. That will force lotus to update the oni FFI dep with the lotus one on updates. Potentially worth understanding if this is going to generate understandable errors if they get out of sync or if there's a better way to help us understand what went wrong

Reply:
Thanks for highlighting! We have to decide if this is a workflow that is okay from this repo or not.

Reply:
Ideally we'd use the submodule from lotus

Reply:
I've added another `go mod replace` so that we use the submodule of `filecoin-ffi` from the `lotus` repo.